storage/state-view/src/account_state_view.rs

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::anyhow;
+use aptos_types::on_chain_config::Version;
+use aptos_types::{
+    access_path::AccessPath,
+    account_config::{
+        AccountResource, BalanceResource, CRSNResource, ChainIdResource, DiemAccountResource,
+    },
+    on_chain_config::{
+        access_path_for_config, dpn_access_path_for_config, ConfigID, ConfigurationResource,
+        OnChainConfig, ValidatorSet,
+    },
+    state_store::state_key::StateKey,
+    validator_config::ValidatorConfigResource,
+};
+use move_core_types::{account_address::AccountAddress, move_resource::MoveResource};
+use serde::de::DeserializeOwned;
+
+pub struct AccountStateView<'a, F>
```

Comment:
This will do, right?
```
pub struct AccountStateView<'a, S> {
    address: AccountAddress,
    state_view: &'a S,
}
```

(`AccountAddress` is `copy`, so we don't really bother to use a ref, but it doesn't matter anyway I guess.)